apps/wallet/src/ui/app/components/menu/content/ledger/ImportLedgerAccounts.tsx

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useFormatCoin } from '@mysten/core';
+import { CheckFill16, LockUnlocked16 as UnlockedLockIcon } from '@mysten/icons';
+import { formatAddress, SUI_TYPE_ARG, type SuiAddress } from '@mysten/sui.js';
+import cl from 'classnames';
+import { useNavigate } from 'react-router-dom';
+
+import { SummaryCard } from '../../../SummaryCard';
+import Overlay from '../../../overlay';
+import { useNextMenuUrl } from '../../hooks';
+import { Button } from '_src/ui/app/shared/ButtonUI';
+import { Link } from '_src/ui/app/shared/Link';
+import { Text } from '_src/ui/app/shared/text';
+
+const mockAccounts = [
+    {
+        isSelected: false,
+        address:
+            '0x7a286c8455a801f6d81faaa0f87543fa4a0de64dcc48b9c9308ee18f0f6ccdd3',
+        balance: 30,
+    },
+    {
+        isSelected: true,
+        address:
+            '0x7a286c8455a401f6d81faaa0f87543fa4a0de64dcc48b9c9308ee18f0f6ccdd3',
+        balance: 30000,
+    },
+];
+
+export function ImportLedgerAccounts() {
+    const accountsUrl = useNextMenuUrl(true, `/accounts`);
+    const navigate = useNavigate();
+
+    return (
+        <Overlay
+            showModal
+            title="Import Accounts"
+            closeOverlay={() => {
+                navigate(accountsUrl);
+            }}
+        >
+            <div className="w-full flex flex-col">
+                <SummaryCard
+                    minimalPadding
+                    header="Connect Ledger Accounts"
+                    body={
+                        <ul className="list-none h-[261px] m-0 p-0 -mr-2 mt-1 py-0 pr-2 overflow-auto custom-scrollbar">
```

Comment:
Never love explicit heights

Reply:
yeah I'm afraid there isn't really a better way to do this AFAIK 😢